integration_tests/snaps/stage_env/snapcraft.yaml

```diff
@@ -0,0 +1,19 @@
+name: test-package
+version: 0.1
+summary: try out $SNAPCRAFT_STAGE
+description: |
+  The cmake-project expects to have a copied directory in the stage directory
+  to consume during its build time.
+
+parts:
+  cmake-project:
+    plugin: cmake
+    source: .
+    configflags:
+        - -DVAR=$SNAPCRAFT_STAGE/copied
```

Comment:
you caught an actual error! I changed var names after testing, but only half way through. I'm fixing this. And since it has a `/` at the end it didn't result in an error